gravitee-apim-portal-webui-next/src/components/pipe/markdown.pipe.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'markdown',
+  standalone: true,
+})
+export class MarkdownPipe implements PipeTransform {
+  transform(value: string): string {
+    if (!value) return value;
+
+    // Replace Markdown headers
+    value = value.replace(/^# (.*$)/gim, '<h1>$1</h1>');
+    value = value.replace(/^## (.*$)/gim, '<h2>$1</h2>');
+    value = value.replace(/^#{3}\s?([^\n]+)/gm, '<h3>$1</h3>');
+    // Replace Markdown links
+    value = value.replace(/\[([^\]]+)\]\(([^)]+)\)/g, '<a href="$2">$1</a>');
```

Comment:
Will you be adding the gravitee link referencing in this PR?

Reply:
Done